exporter/kafkaexporter/factory.go

```diff
@@ -72,8 +75,10 @@ func createDefaultConfig() configmodels.Exporter {
 		RetrySettings:   exporterhelper.CreateDefaultRetrySettings(),
 		QueueSettings:   exporterhelper.CreateDefaultQueueSettings(),
 		Brokers:         []string{defaultBroker},
-		Topic:           defaultTopic,
-		Encoding:        defaultEncoding,
+		// 'otlp_spans' is being deprecated as the default topic.
```

Comment:
we are not deprecating the topic name right? we want to use different topic names for different signals

Reply:
@bogdandrutu this brings an interesting point. Maybe the factory should be capable of creating different defaults for different signals.

Reply:
@pavolloffay true, we should file an issue to discuss this.

Reply:
I'll update the comment

Reply:
@bogdandrutu created https://github.com/open-telemetry/opentelemetry-collector/issues/2160